### ISSUE TITLE
Install Recommended Cypress Plugins

### ISSUE THREAD

**Issue by jsmith-obrien** [jsmith-obrien/somersetrah-tests#3]
Given I want better testing support and utility functions
When I run:
  `npm install --save-dev cypress-real-events`
  `npm install --save-dev cypress-axe`
Then real browser events and accessibility checks should be available

**Comment by jsmith-obrien** [comment]
found 0 vulnerabilities
jobrien80@Joshuas-MacBook-Air SomersetRAH % cd cypress
jobrien80@Joshuas-MacBook-Air cypress %  npm install --save-dev cypress-real-events

up to date, audited 180 packages in 371ms

41 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
jobrien80@Joshuas-MacBook-Air cypress %  npm install --save-dev cypress-axe

added 2 packages, and audited 182 packages in 482ms

41 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
jobrien80@Joshuas-MacBook-Air cypress %
